ui/src/app/kvm/components/KVMHeaderForms/AddKVM/AddLxd/AddLxd.test.tsx

```diff
@@ -68,14 +68,11 @@ describe("AddLxd", () => {
         <MemoryRouter
           initialEntries={[{ pathname: "/kvm/add", key: "testKey" }]}
         >
-          <AddLxd setKvmType={jest.fn()} />
+          <AddLxd clearHeaderContent={jest.fn()} setKvmType={jest.fn()} />
         </MemoryRouter>
       </Provider>
     );
 
-    expect(wrapper.find("[data-test='step-number']").text()).toBe(
-      "Step 1 of 2"
```

Comment:
This will be replaced in https://github.com/canonical-web-and-design/app-squad/issues/249